src/cct.c

```diff
@@ -268,14 +268,16 @@ int main(int argc, char **argv) {
         int dec = atoi (opt_arg (o, "d"));
         decimals_angles = dec;
         decimals_distances = dec;
-        nfields--;
     }
 
     if (opt_given (o, "s")) {
         skip_lines = atoi (opt_arg(o, "s"));
     }
 
     if (opt_given (o, "c")) {
+        /* reset colum numbers to ease comment output later on */
+        for (i=0; i<4; i++) columns_xyzt[i] = 0;
```

Comment:
For style coherency this should be

    for (i=0; i<4; i++)
        columns_xyzt[i] = 0;